### PR TITLE
Formatted markdown message8

### PR DESCRIPTION
**Pull request by jopaleti**
#### Ticket Link

```release-note
NONE
```

### CODE REVIEW

File: server/.golangci.yml
Comment (M-ZubairAhmed) [inline server/.golangci.yml:1]: please revert not related change

File: server/channels/api4/ip_filtering_test.go
Comment (M-ZubairAhmed) [inline server/channels/api4/ip_filtering_test.go:1]: please revert not related change

File: webapp/package-lock.json
Comment (M-ZubairAhmed) [inline webapp/package-lock.json:1]: please revert not related change

File: webapp/package.json
Comment (M-ZubairAhmed) [inline webapp/package.json:1]: please revert not related change

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
please add the JIRA and Github ticket in description

File: webapp/package.json
Comment (jopaleti) [reply webapp/package.json:1]: Sorry for that, I will re submit it now

**Comment by jopaleti** [comment]
@M-ZubairAhmed I have submitted a new PR on [https://github.com/mattermost/mattermost/pull/28655](https://github.com/mattermost/mattermost/pull/28655)

**Pull request closed without merging**